### ISSUE TITLE
Enhancement: Adding mobile responsiveness

### ISSUE THREAD

**Issue by JhoanAlvear** [whtrapp/whtrapp.github.io#8]
The application is not responsive, let me add boostrap and make it responsive, assign me this task
![image](https://github.com/user-attachments/assets/5a2ebd07-5535-4e94-80c2-c287912b4540)

**Comment by AalokeCode** [comment]
Hey! Upon initial discussions, we've decided not to add Bootstrap cause it introduces a lot of unnecessary classes and is inefficient. So if you want to make it responsive using plain CSS or suggest a better approach, I'd be happy to assign you the issue.

**Comment by JhoanAlvear** [comment]
ok, i can use  Meta tag for optimize screen smalls

**Comment by Anurag-singh-thakur** [comment]
@AalokeCode Hey i can make it responsive very soon using the plain CSS please assign me this task ..

**Comment by AalokeCode** [comment]
Alright! I'm assigning you guys the issue. We will check the PR's and merge what looks good.

**Comment by Neelesh19634** [comment]
@AalokeCode Hey i can make it responsive very soon using the  CSS please assign me this task ..

**Comment by Anurag-singh-thakur** [comment]
![image](https://github.com/user-attachments/assets/9be34f06-f1e8-434c-96e4-1940e8e2a71a) [mobiles ]
![image](https://github.com/user-attachments/assets/be375f48-690f-47f3-8548-321fc7344d41)[tablets]
 Here is the look for the website . i am making a PR please merge it under the hactoberfest.

**Comment by AalokeCode** [comment]
@Neelesh19634 There are already 2 PR's working on this. So I can't assign you the issue

**Comment by Anurag-singh-thakur** [comment]
![image](https://github.com/user-attachments/assets/90e086b8-c716-406e-bc49-45f1dfb7a963)
![image](https://github.com/user-attachments/assets/86e445bd-80f3-4475-b967-5722276d66b5)
![image](https://github.com/user-attachments/assets/c19a75c9-fee1-4d68-ac60-04e703bebf16)
attached multiple screenshots there is no problem @AalokeCode  brother...please merge them...i don't know why there is an issue at your side

**Comment by AalokeCode** [comment]
Hey @Anurag-singh-thakur. Please don't modify the design that much. Please keep it intact to the original design. Request you to modify the code accordingly :)

**Comment by Anurag-singh-thakur** [comment]
> Hey @Anurag-singh-thakur. Please don't modify the design that much. Please keep it intact to the original design. Request you to modify the code accordingly :)

brother for mobiles you need to modify designs as they don't have that much space like monitor or screens to keep the pages intact

**Comment by AalokeCode** [comment]
Yeah I agree. But you have removed most of the text and features which changed the website a lot.

**Comment by AalokeCode** [comment]
Hey @JhoanAlvear! Can I get an update on this issue?

**Comment by ahmedrazabaloch** [comment]
@AalokeCode please assign it to me then I will start working on it
